### DISCUSSION TITLE
Convert JPG to DICOM

### DISCUSSION THREAD

**Discussion by datnk-hust** [nextgenhealthcare/connect#4970] [Q&A]
Hi everyone,
I'm working with Mirth version 3.11 and I face converting nondicom image to dicom.
Can someone suggest me. Any support is greatly appreciated. Thank for all!

**Comment by jonbartels** [comment]
You should try a few searches first:

- MC Forums have been around for almost 20 years and have a lot of historical information - https://forums.mirthproject.io/
- https://github.com/nextgenhealthcare/connect/search?q=jpg+dicom&type=issues may have some related issues, search discussions also

After that can you show what you have already tried?

**Comment by montamrta** [comment]
You can try another DICOM anonymizer tool, perhaps Adage? https://www.ai-medical.ch/adage

**Comment by jmiddleton** [comment]
I’ve been trying to transcode a dicom to jpeg and there is no documentation or forum that explains how to do it. 

Is there any example somewhere?
